src/app.css

```
@@ -60,3 +60,60 @@
+
+.tiptap-content blockquote {
+	margin: 0.75rem 0;
+	padding-left: 0.9rem;
+	border-left: 3px solid color-mix(in srgb, currentColor 18%, transparent);
+	opacity: 0.85;
+}
+
+.tiptap-content :not(pre) > code,
+.tiptap-content li > code,
+.tiptap-content p > code {
+	font-family: ui-monospace, SFMono-Regular, SFMono-Regular, Consolas, 'Liberation Mono', Menlo,
```

Stylelint (17.7.0)

[error] Duplicate font-family name "SFMono-Regular" (font-family-no-duplicate-names)

(font-family-no-duplicate-names)